sdks/python/apache_beam/internal/gcp/auth.py

```diff
@@ -59,16 +73,19 @@ def set_running_in_gce(worker_executing_project):
   executing_project = worker_executing_project
 
 
-def get_service_credentials():
+def get_service_credentials(pipeline_options):
   """For internal use only; no backwards-compatibility guarantees.
 
   Get credentials to access Google services.
+  Args:
+    pipeline_options: Pipeline options, used in creating credentials
```

Comment:
I'd say something here like: 

Pipeline options. Required on initial credential initialization when the impersonate_service_account option is provided, otherwise passing `None` is acceptable. 

You can do it in a follow up change